garnett/managers.py

```diff
@@ -0,0 +1,6 @@
+from django.db.models import QuerySet
```

Comment:
BookQueryset should be in /tests/library

Reply:
Can do